fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfLexicon.java

```diff
@@ -262,6 +260,18 @@
      */
     public static final String NT_VERSION_FILE = "nt:versionFile";
 
+    // VERSIONING
+    /**
+     * This is a holder for an eventually Memento TimeMap RDF type.
+     */
+    public static final String VERSIONING_TIMEMAP_TYPE = REPOSITORY_NAMESPACE + "TimeMap";
+
+    /**
+     * This is an internal RDF type for versionable resources, this may be replaced by a Memento type.
+     */
+    public static final Property VERSIONED_RESOURCE =
```

Comment:
Actually, the namespace here should be:
`http://fedora.info/definitions/fcrepo#VersionedResource`
..instead of:
`http://fedora.info/definitions/v4/repository#VersionedResource`